### DISCUSSION TITLE
Ecto prefix support

### DISCUSSION THREAD

**Discussion by kraegpoeth** [naymspace/backpex#621] [Ideas]
With the brilliant ecto prefix, I hope that the maintainers/creators of Backpex would consider supporting it as well. I would enable plug n' play admin panel for multi tenant apps, such as those built with Triplex.
I feel like it would be pretty straight forward to just add an optional callback or setting to the Backpex macro, with a prefix. Then all repo calls just need to add that.

I managed to make it work for index, and show actions via the item_query, where i add the prefix, but unfortunately the Create Update, Delete queries does not run through that callback.

**Comment by pehbehbeh** [comment]
I am currently working on an adapter (Ecto, Ash, ...) abstraction for Backpex. Once this is done, prefix support can easily be added to the `Backpex.Adapters.Ecto`.
